google-account-plugin/src/com/google/cloud/tools/intellij/stats/TrackingEventBuilder.java

```diff
@@ -39,19 +38,26 @@
   }
 
   @Override
-  public FluentTrackingEventWithValue withLabel(String label) {
+  public FluentTrackingEventWithMetadata withLabel(String label) {
     this.label = Preconditions.checkNotNull(label);
     return this;
   }
 
   @Override
-  public PingsAnalytics setValue(Integer value) {
+  public FluentTrackingEventWithMetadata withLabel(String label, int value) {
```

Comment:
How about changing the `int` type of `value` to `String` instead of adding a new method `withMessage()`? That is, the metadata pair will be added as `<label>=<value>` whenever this method is called. You can then call `withLabel("message", errorMessage)`.